app/models/list.rb

```diff
@@ -1,5 +1,5 @@
 class List < ApplicationRecord
-  has_many :entries, dependent :destroy
+  has_many :entries, dependent: :destroy
   belongs_to :user
-  validates :list_name, length: { maximum: 140 }
+  validates :title, length: { maximum: 140 }
```

Comment:
I needed to make this change in order to get tests working.

In your schema.rb and databases, you have a title field which I believe you intended to use but list_name is just a variable.  So I changed this to match what I think you're intention was so I could write a test around the validation.